File: Arcade/Intro/1 - The Journey Begins/3 - checkPalindrome/checkPalindrome.cpp
bool checkPalindrome(std::string inputString) {
    
    for(int i = 0; i < inputString.size()-1; i++)
    {
        char charLeftToRight = inputString.at(i);
        char charRightToLeft = inputString.at(inputString.size()-i-1);
        bool charsAreDifferent = charLeftToRight != charRightToLeft;
            
        if(charsAreDifferent) return false;
    }
    
    return true;
}

File: Arcade/Intro/1 - The Journey Begins/3 - checkPalindrome/checkPalindrome.java
boolean checkPalindrome(String inputString) {

    for(int i = 0; i < inputString.length(); i++)
    {
        char charLeftToRight = inputString.charAt(i);
        char charRightToLeft = inputString.charAt(inputString.length()-i-1);
        boolean charsAreDifferent = charLeftToRight != charRightToLeft;
        
        if(charsAreDifferent) return false;
    }
    return true;
}
